RGBMatrixEmulator/adapters/sixel_adapter.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+import io
+import libsixel as sixel
+
+from PIL import Image, ImageDraw, ImageEnhance
+from typing import List
+
+from RGBMatrixEmulator.adapters.base import BaseAdapter
+from RGBMatrixEmulator.graphics.color import Color
+
+
+def enlarge_pixels(pixels: List[List[Color]], scale: int, outline: int) -> Image.Image:
```

Comment:
I would prefer these match the other adapters -- methods on the adapter class itself, prefixed by double underscore (i.e. `def __enlarge_pixels(. . .)`)

Unless there's a good reason not to?

Reply:
While I'm not strongly biased to this, I think we can move the `enlarge_pixel` function to a module. That way the same drawing function can be used by both sixel and browser adapter. Perhaps in a different PR -- and in which case I am ready to move it in the class to match the codebase style. Let me know what you prefer!

Reply:
Ok, that makes sense. I don't have issue with it as-is then